src/PowerShellEditorServices/Console/ConsoleReadLine.cs

```diff
@@ -20,12 +20,28 @@ namespace Microsoft.PowerShell.EditorServices.Console
     internal class ConsoleReadLine
     {
         #region Private Field
+        private static IConsoleOperations s_consoleProxy;
 
         private PowerShellContext powerShellContext;
 
         #endregion
 
         #region Constructors
+        static ConsoleReadLine()
+        {
+            // Maybe we should just include the RuntimeInformation package for FullCLR?
```

Comment:
I don't know the answer to this question but I think we should answer it in this PR.

Reply:
I'm split on it.  The compiler directives feel a little messy, especially if we end up using them more.  On the other hand including an assembly that's probably going to be included in a whole bunch of modules doesn't feel great either.

Reply:
I opted to include `System.Runtime.InteropServices.RuntimeInformation.dll` in `PSReadLine` so I could build and ship a single assembly that works on all supported platforms - it definitely kept builds/packing simpler.

Reply:
@lzybkr Yeah that would be ideal for sure.  Do you know of a way to do that and remain buildable on Unix?  I came up empty last time I looked into it. The blocker was targeting only `net46x` required Windows (or Mono?) and targeting only `netstandard` required shims to load properly in Windows PowerShell.

Reply:
@SeeminglyScience - I haven't done this, but I think you would need to build targets, one for `net46x` and one for `netstandard`. I don't run `PSReadLine` tests on Linux/Mac (and there is very little reason to, the platform specifics are mostly mocked anyway), so haven't dealt with this situation yet.